sdk/web-pubsub/web-pubsub/src/groupClient.ts

```diff
@@ -0,0 +1,305 @@
+    this.hubName = hubName;
+  }
+  /**
+   * Add a specific connection to this group
+   *
+   * @param connectionId The connection id to add to this group
+   * @param options Additional options
+   */
+  public async addConnection(
+    connectionId: string,
+    options: GroupAddConnectionOptions = {}
+  ): Promise<RestResponse> {
+    const { span, updatedOptions } = createSpan(
+      "WebPubSubManagementClient-group-addConnection",
+      options
+    );
+
+    try {
+      const res = await this.client.webPubSub.addConnectionToGroup(
+        this.hubName,
+        this.groupName,
+        connectionId,
+        updatedOptions
+      );
+
+      if (res._response.status === 404) {
+        throw new RestError(
+          `Connection id '${connectionId}' doesn't exist`,
+          undefined,
+          res._response.status,
+          res._response.request,
+          res._response
+        );
+      }
+
+      return res;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Remove a specific connection from this group
+   *
+   * @param connectionId The connection id to remove from this group
+   * @param options Additional options
+   */
+  public async removeConnection(
+    connectionId: string,
+    options: GroupRemoveConnectionOptions = {}
+  ): Promise<RestResponse> {
+    const { span, updatedOptions } = createSpan(
+      "WebPubSubManagementClient-group-removeUser",
+      options
+    );
+
+    try {
+      const res = await this.client.webPubSub.removeConnectionFromGroup(
+        this.hubName,
+        this.groupName,
+        connectionId,
+        updatedOptions
+      );
+
+      return res;
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Add a user to this group
+   *
+   * @param username The user name to add
+   * @param options Additional options
+   */
+  public addUser(username: string, options: GroupAddUserOptions = {}): Promise<RestResponse> {
+    const { span, updatedOptions } = createSpan("WebPubSubManagementClient-group-addUser", options);
+
+    try {
+      return this.client.webPubSub.addUserToGroup(
+        this.hubName,
+        this.groupName,
+        username,
+        updatedOptions
+      );
+    } finally {
+      span.end();
+    }
+  }
+
+  /**
+   * Check if a user is in this group
+   *
+   * @param username The user name to check for
+   * @param options Additional options
+   */
+  public async hasUser(username: string, options: GroupHasUserOptions = {}): Promise<boolean> {
+    const { span, updatedOptions } = createSpan("WebPubSubManagementClient-group-hasUser", options);
+
+    try {
+      const res = await this.client.webPubSub.userExistsInGroup(
+        this.hubName,
+        this.groupName,
+        username,
+        updatedOptions
+      );
+
+      if (res._response.status === 200) {
+        return true;
+      } else if (res._response.status === 404) {
+        return false;
+      } else {
+        // this is sad - wish this was handled by autorest.
```

Comment:
it can be, at least for non success http codes! I think you might just need to tweak the swagger